pkg/apis/kops/v1alpha1/defaults.go

```diff
@@ -17,9 +17,48 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"github.com/golang/glog"
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
 func addDefaultingFuncs(scheme *runtime.Scheme) error {
-	return nil
+	RegisterDefaults(scheme)
+	return scheme.AddDefaultingFuncs(
+		SetDefaults_ClusterSpec,
+	)
+}
+
+func SetDefaults_ClusterSpec(obj *ClusterSpec) {
+	if obj.Topology == nil {
+		obj.Topology = &TopologySpec{}
+	}
+
+	if obj.Topology.Masters == "" {
+		obj.Topology.Masters = TopologyPublic
```

Comment:
I don't think anything has changed here ... we always allowed that configuration.